### DISCUSSION TITLE
基于fastapi重构的后端项目能否放出

### DISCUSSION THREAD

**Discussion by xyzhou-1** [521xueweihan/HelloGitHub#1645] [提问]
原后端项目长时间没有更新，所以想问基于fastapi重构的后端项目能否放出。hellogithub帮助我发现了很多有用的项目，想着也出份力。有简单的flask经验，但都是些学校大作业的玩具项目，希望不要介意，我会尽力学习的（大作业的前后端是自己一个星期自学完成的，脑子一般，但是喜欢折腾）。

**Comment by 521xueweihan** [comment]
等我！
